File: quests/quest00/ex00.py
"""ex00 - my_first_file must simply exist. 1 point."""

FILE = "my_first_file"


def run(g):
    g.check("%s exists and is a regular file" % FILE,
            g.is_file(FILE),
            "" if g.exists(FILE) else "no such file in this directory")

File: quests/quest00/ex00.js
"use strict";
// my_first_file must simply exist. 1 point.

const FILE = "my_first_file";

exports.run = function (g) {
  g.check(
    `${FILE} exists and is a regular file`,
    g.isFile(FILE),
    g.exists(FILE) ? "" : "no such file in this directory"
  );
};
